src/components/atoms/UiLink/UiLink.vue

```diff
@@ -104,21 +105,8 @@ const {
     --icon-color: #{functions.var($element + "-icon", color, var(--color-icon-primary))};
 
     flex: none;
-    margin: functions.var($element + "-icon", margin, 0 var(--space-4) 0 0);
```

Comment:
we can't use flex and we should use margin instead gap :( sorry. You can keep this approach for `ui-button--text` because it isn't be place in text